nodejs/examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Examples
+
+This folder contains several examples that demonstrate how to use onnxruntime Node.js binding.
+
+## Examples List
+
+### [01 Basic Usage](./01_basic-usage/README.md)
+
+This example is a demonstration of basic usage of ONNX Runtime Node.js binding.
+
+### [02 Create Tensor](./02_create-tensor/README.md)
+
+This example is a demonstration of basic usage of creating tensors.
+
+<!--
```

Comment:
Is there a reason why this is commented out ?

Reply:
the actual examples are not ready. tensor utils will be added in future